Scripts/script-TaniumAskQuestionComplex.yml

```diff
@@ -165,3 +165,4 @@ dependson:
   - tn-result-info
   - tn-add-question-complex
 timeout: 0s
+releaseNotes: Deprecated script
```

Comment:
please write the command that replace this script.
Deprecated script, use `command name` instead.

Reply:
No need for that one anymore, the new tn-ask-question supports plain english, so it already supports complex questions